api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -109,6 +109,21 @@ message LatencyFault {
   FaultLatencyDuration latency_duration = 2 [ (validate.rules).message.required = true ];
 }
 
+// Enforce faults on upstream redis cluster.
+message RedisFaultTargeting {
+  // A downstream responsible for enforcing a fault. Only requests that are started
+  // by the specified downstream are considered for a fault injection.
+  oneof downstream_type {
+    option (validate.required) = true;
+
+    // A single downstream cluster.
+    SingleCluster downstream_cluster = 1;
```

Comment:
Thinking about this some more don't we also need to specify which upstream redis cluster we're targeting for faults? Not sure what kind of flexibility we have here atm

Reply:
Yeah I got rid of this from the last iteration, but yes we do need both clusters defined.